src/Tribe/Tickets.php

```diff
@@ -3590,8 +3590,17 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 				return;
 			}
 
-			/** @var \Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
-			$attendee_registration = tribe( 'tickets.attendee_registration' );
+			/**
+			 * This Try/Catch is present to deal with a problem on Autoloading from version 5.1.0 ET+ with ET 5.0.3.
+			 *
+			 * @todo Needs to be revised once proper autoloading rules are done for Common, ET and ET+.
+			 */
+			try {
+				/** @var \Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
+				$attendee_registration = tribe( 'tickets.attendee_registration' );
+			} catch( RuntimeException $error ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after CATCH keyword; 0 found

<sub>hash: c2c900740aad6902e13fce64f763bd32</sub>